Failure at compilation

```bash
In file included from /home/victor/code/catkin_workspace/src/descartes/descartes_planner/src/planning_graph.cpp:28:0:
/home/victor/code/catkin_workspace/src/descartes/descartes_planner/include/descartes_planner/planning_graph_edge_policy.h: In member function ‘void descartes_planner::DefaultEdgesWithTime::consider(const double*, const double*, std::size_t)’:
/home/victor/code/catkin_workspace/src/descartes/descartes_planner/include/descartes_planner/planning_graph_edge_policy.h:37:17: error: ‘accumulate’ is not a member of ‘std’
     auto cost = std::accumulate(delta_buffer_.cbegin(), delta_buffer_.cend(), 0.0);
                 ^~~
descartes/descartes_planner/CMakeFiles/descartes_planner.dir/build.make:62 : la recette pour la cible « descartes/descartes_planner/CMakeFiles/descartes_planner.dir/src/planning_graph.cpp.o » a échouée
make[2]: *** [descartes/descartes_planner/CMakeFiles/descartes_planner.dir/src/planning_graph.cpp.o] Erreur 1
make[2]: *** Attente des tâches non terminées....
```

```bash
g++ --version
g++ (Ubuntu/Linaro 6.3.0-18ubuntu2~16.04) 6.3.0 20170519
Copyright (C) 2016 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

--- a/descartes_planner/include/descartes_planner/planning_graph_edge_policy.h
+++ b/descartes_planner/include/descartes_planner/planning_graph_edge_policy.h
@@ -2,6 +2,7 @@
 #define PLANNING_GRAPH_EDGE_POLICY_H
 
 #include "descartes_planner/ladder_graph.h"
+#include <numeric>
 
 namespace descartes_planner
 {

